apps/dashboard/lib/auth/types.ts

```diff
@@ -14,6 +14,7 @@ export interface User {
   lastName: string | null;
   avatarUrl: string | null;
   fullName: string | null;
+  impersonator: object | undefined;
```

Comment:
nit:
I'm surprised biome let this one pass, `unknown` should be what we use

Reply:
Why would change what is returned from workos?

Reply:
I just mapped 1:1 auth response